server/main.cpp

```diff
@@ -0,0 +1,51 @@
+#include "main.h"
+#include "global.h"
+#include "server.h"
+
+#define DEFAULT_IP "127.0.0.1"
+#define DEFAULT_PORT "3306"
+#define MAXLINE 100
+
+bool bRunning = true;
+
+int main(int argc, char **argv) {
+
+  char arg1[20], arg2[6];
```

Comment:
As far as I remember we did some refactoring already to this part in your other branch. I want to ask you to copy those changes and paste it here. This way I can see a more clear picture of how we handle the params.